x-pack/legacy/plugins/siem/public/components/ml/permissions/has_ml_user_permissions.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { MlCapabilities } from '../types';
+
+export const hasMlUserPermissions = (capabilities: MlCapabilities): boolean => {
+  const canRead =
+    capabilities.isPlatinumOrTrialLicense &&
+    capabilities.mlFeatureEnabledInSpace &&
```

Comment:
checking for `isPlatinumOrTrialLicense` and `mlFeatureEnabledInSpace` probably isn't needed here. if either are `false` then standard permissions like `canGetJobs` will be false.
The flags `upgradeInProgress`, `isPlatinumOrTrialLicense` and `mlFeatureEnabledInSpace` are there for additional information in case you need a reason for why the capabilities are disabled.

Reply:
Got it, I will keep the plumbing for the API endpoint but remove the explicit check here for those two features.